addons/l10n_lb/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Lebanon - Accounting",
+    "countries": ["lb"],
+    "version": "1.0",
+    "description": """
+This is the base module to manage the accounting chart for Lebanon in Odoo.
```

Comment:
I think you need to add this description (and probably the name too) to the .pot in the module base, so it can be translated later on Transifex